.github/workflows/ci.yml

```diff
@@ -165,6 +165,21 @@ jobs:
               ../build/examples/trixi_controller_data_f . libelixir_p4est2d_dgsem_euler_sedov.jl
         env:
           LIBTRIXI_DEBUG: all
+      - name: Run julia tests with coverage
+        if: ${{ matrix.test_type == 'coverage' }}
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: true
+          project: ./libtrixi-julia
```

Comment:
This should IMHO point to the LibTrixi.jl directory, not the runtime dir we created.

Reply:
:pray: